ciao-scheduler/scheduler.go

```diff
@@ -697,10 +709,10 @@ func startWorkload(sched *ssntpSchedulerServer, controllerUUID string, payload [
 
 	var targetNode *nodeStat
 
-	if workload.networkNode == 0 {
-		targetNode = pickComputeNode(sched, controllerUUID, &workload)
-	} else { //workload.network_node == 1
+	if workload.networkNode {
 		targetNode = sched.pickNetworkNode(controllerUUID, &workload)
+	} else { //workload.network_node == false
```

Comment:
@tpepper un-related to this patch. But why do we have a method in one place and function in the other?